src/main/java/org/xstefank/verification/RepositoryFormatVerification.java

```diff
@@ -0,0 +1,17 @@
+package org.xstefank.verification;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public class RepositoryFormatVerification implements Verification {
+
+    private final String MESSAGE = "Bad repository format in configuration file";
+    private final String REGEX = "^[A-z,0-9]*./[A-z,0-9,-].*$";
+
+    @Override
+    public void verify(FormatConfig formatConfig) throws BadFormatException {
+        String repoFormat = formatConfig.getRepository();
+        if (!repoFormat.matches(REGEX) || repoFormat.split("/").length != 2) {
```

Comment:
I am not sure `|| repoFormat.split("/").length != 2` is required

Reply:
@lukasburda try to type `test/slash-repo` as a name of the new repository at GitHub ;)

Reply:
but I see now that you want to check whether what the user typed is a valid repository, still I am not convinced that this check is necessary and it cannot be caught directly by regex